docker-testing/Dockerfile-fetchers

```
@@ -0,0 +1,17 @@
+FROM netmeld-test-base
+
+ENV NETMELD_DB netmeld_test_fetchers
+
+WORKDIR /home/netmeld/setup
+
+RUN apt install --quiet --quiet --assume-yes --option Dpkg::Use-Pty=0 \
+      --no-install-recommends --no-install-suggests \
+      ./netmeld-*core.deb ./netmeld-*fetchers.deb ./netmeld-*tool-clw.deb
+
+WORKDIR /home/netmeld
+
+RUN rm -rf ./setup
+
+WORKDIR /home/netmeld/contents/fetchers
+
+CMD ["./test.py"]
```

Since we aren't using for testing, it should drop to a shell or execute a command.  Furthermore, we should confine it to the `netmeld` user.  So something like:
```suggestion
USER netmeld

CMD ["/bin/bash"]
```